integration-tests/opentelemetry-jdbc-instrumentation/README.md

```diff
@@ -0,0 +1,37 @@
+# OpenTelemetry JDBC instrumentation example
+
+## Running the tests
+
+
+To run the tests in a standard JVM with an Oracle, PostgreSQL and MariaDB databases started as a Docker containers, you can run the following command:
+
+```
+mvn verify -Dtest-containers -Dstart-containers
+```
+
+To also test as a native image, add `-Dnative`:
+
+```
+mvn verify -Dtest-containers -Dstart-containers -Dnative
+```
+
+You can also run tests with a specific database image
```

Comment:
Delete this line

Reply:
I completely missed it @gastaldi , thank you.